### PR TITLE
chore: parametrize user image docker repository

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

closes FL-1541

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: backend/substrapp/compute_tasks/image_builder.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 IMAGE_BUILD_CHECK_DELAY = settings.IMAGE_BUILD_CHECK_DELAY
 REGISTRY = settings.REGISTRY
 SUBTUPLE_TMP_DIR = settings.SUBTUPLE_TMP_DIR
+USER_IMAGE_REPOSITORY = settings.USER_IMAGE_REPOSITORY
```
Comment (SdgJlbl) [inline backend/substrapp/compute_tasks/image_builder.py:19]: Unrelated comment but we really need to rename that module at some point 😅

File: examples/values/backend-org-1.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ containerRegistry:
     - image: substra/substra-tools:latest
       sourceRegistry: ghcr.io
       dstImage: substra/substra-tools:latest
-
+  userImageRepository: substra-user-images
```
Comment (SdgJlbl) [inline examples/values/backend-org-1.yaml:45]: I think we want to add the parameter explicitly in all orgs, for clarity.

**Review by SdgJlbl — APPROVED** [review]
Minor comment but otherwise LGTM 👍 
Thanks!

File: examples/values/backend-org-1.yaml (hunk 2 above)
Comment (ThibaultFy) [reply examples/values/backend-org-1.yaml:45]: I also changed the default to `substra/user-image` to remove the `substrafoundation` mention out of date. But I let the org1 to `substra-user-images` to continue to test that the feature works

**Pull request merged**